tests/test_alerter.py

```diff
@@ -254,40 +258,40 @@ def test_should_not_alert_ooh(self):
             self.assertEqual(a.should_alert(m), alerter.AlertType.NONE)
             self.assertEqual(a._ooh_failures, ["fail"])
 
-    @pytest.mark.skip("TZ test logic need fixing")
+    @pytest.mark.skip("TZ test logic needs fixing")
+    @freeze_time("2020-03-10 10:30", tz_offset=LOCAL_TZ_OFFSET)
     def test_should_alert_ooh(self):
         config = {"times_type": "only", "time_lower": "10:00", "time_upper": "11:00"}
         a = alerter.Alerter(config)
         m = monitor.MonitorFail("fail", {})
         m.run_test()
-        with freeze_time("2020-03-10 10:30", tz_offset=self.utcoffset):
-            self.assertEqual(a.should_alert(m), alerter.AlertType.FAILURE)
-            self.assertEqual(a._ooh_failures, [])
+        self.assertEqual(a.should_alert(m), alerter.AlertType.FAILURE)
+        self.assertEqual(a._ooh_failures, [])
 
-    @pytest.mark.skip("TZ test logic need fixing")
+    @freeze_time("2020-03-10 10:30", tz_offset=LOCAL_TZ_OFFSET)
     def test_should_alert_limit(self):
         config = {
             "times_type": "only",
             "time_lower": "10:00",
             "time_upper": "11:00",
             "limit": 2,
+            "times_tz": "+07:00",
```

Comment:
I'm not 100% sure if this is the right / desired logic. I think since we're just testing limits, as long as `times_tz` is set, and the offset is right, that's sufficient to test the behavior you're trying to test here?